packages/devtools/src/SlotCSSRules.tsx

```diff
@@ -16,24 +16,39 @@ const useStyles = makeStyles({
     ...shorthands.borderBottom('1px', 'solid', tokens.slotNameBorder),
     backgroundColor: tokens.slotNameBackground,
   },
+  slotNameCollapsed: {
```

Comment:
```ts
const useStyles = makeStyles({
  slotName: {
    display: 'flex',
    alignItems: 'center',
    position: 'relative',

    cursor: 'pointer',
    ...shorthands.padding('2px', '5px'),
    ...shorthands.borderTop('1px', 'solid', tokens.slotNameBorder),
    ...shorthands.borderBottom('1px', 'solid', tokens.slotNameBorder),
    backgroundColor: tokens.slotNameBackground,

    ':before': {
      content: "''",
      position: 'absolute',
      right: '5px',
      width: 0,
      height: 0,
      fontSize: 0,
      ...shorthands.borderTop('3px', 'solid', tokens.foreground),
      ...shorthands.borderRight('3px', 'solid', 'transparent'),
      ...shorthands.borderLeft('3px', 'solid', 'transparent'),
      ...shorthands.borderBottom('3px', 'solid', 'transparent'),
    },
  },
  slotNameCollapsed: {
    ':before': {
      ...shorthands.borderTop('3px', 'solid', 'transparent'),
      ...shorthands.borderRight('3px', 'solid', tokens.foreground),
    },
  },
  rules: {
    ...shorthands.padding(0, '5px'),
  },
});
```

This snippet will make it better 😉

Reply:
Ahh I see I like it much better :) applied